docs/developer/architecture.rst

```diff
@@ -316,6 +316,7 @@ by documentation for now, the rest is coming soon...).
    │   │   ├───sql
    │   │   └─── :doc:`xgboost </flow/modin/experimental/xgboost>`
    │   ├───pandas
+   │   │   ├─── :doc:`dataframe </flow/modin/pandas/dataframe>`
```

Comment:
I do not see any line related to generating documentation of the methods from docstrings in `docs/flow/modin/pandas/dataframe.rst`. The same is true for #3056. Why are the lines related to generating documentation missed? We should probably add those and insert links to their API in API section of the Modin documentation ([link](https://modin.readthedocs.io/en/latest/developer/architecture.html#api)).

Reply:
Ok, `DataFrame` class docstring was added, but i don't think we should add methods docstrings since Modin and pandas `DataFrame` methods API are identical and we can just mention this in documentation with link to pandas.

Reply:
Yes, we do not need to add those method docstrings.